package.json

```diff
@@ -94,6 +94,7 @@
     "@octokit/auth-app": "6.0.3",
     "@octokit/core": "5.0.2",
     "@percy/cli": "1.27.4",
+    "@reach/dialog": "0.10.5",
```

Comment:
for whatever reason this package is no longer getting hoisted. So when the runner tries to build the reporter, the package is missing. This puts it in the root of the repo so its available in both places as opposed to installing it in 2 places